Source/santad/EventProviders/SNTEndpointSecurityManager.mm

```diff
@@ -127,94 +149,103 @@ - (void)messageHandler:(const es_message_t *)m API_AVAILABLE(macos(10.15)) {
                                                encoding:NSUTF8StringEncoding]];
       }
       sm.args_array = (void *)CFBridgingRetain(args);
-
       callback = self.logCallback;
-      audit_token = m->event.exec.target->audit_token;
-      sm.ppid = m->event.exec.target->original_ppid;
-
-      size_t l = m->event.exec.target->executable->path.length;
-      strncpy(sm.path,m->event.exec.target->executable->path.data, l);
-      sm.path[l] = '\0';
-
       break;
     }
     case ES_EVENT_TYPE_NOTIFY_CLOSE: {
-      if (!m->event.close.modified) return;
+      if (!m->event.close.modified) {
+        es_free_message(m);
+        return;
+      }
```

Comment:
This path will never be hit, is it just a last ditch check?

Reply:
Removed